test/webhook-apicoverage/tools/tools.go

```diff
@@ -229,3 +236,15 @@ func WriteResourcePercentages(outputFile string,
 
 	return ioutil.WriteFile(outputFile, []byte(htmlData), 0400)
 }
+
+// Helper function to cleanup any existing Junit XML files.
+// This is done to ensure that we only have one Junit XML file providing the
+// API Coverage summary.
+func CleanupJunitFiles(artifactsDir string) {
```

Comment:
Can we make this function internal and call it before/after we create the Junit XML file for API Coverage? Maybe here https://github.com/knative/pkg/blob/a1b3961a1d0badabee1e5e0fd3a18c3fbace4162/test/webhook-apicoverage/tools/tools.go#L223 ?

Reply:
That be forcing all repos to follow a pattern. Its better to have it as composable rather than forcing repos to follow one, so I would keep this way.